packages/@sanity/base/sanity.json

```diff
@@ -50,14 +50,6 @@
       "description": "Function that generates or modifies asset URLs for display in various contexts",
       "path": "assets/asset-url-builder"
     },
-    {
-      "name": "part:@sanity/base/search",
-      "description": "Parse the search query and fetch search results from the dataset"
-    },
-    {
-      "name": "part:@sanity/base/search/weighted",
-      "description": "Weighted search implementation"
-    },
```

Comment:
Technically speaking, this is a breaking change. I'm not sure if anyone is likely to have actually used these though - but there is a reference to them from this shared content demo, for instance: https://github.com/runeb/shared-content-demo/blob/main/schemas/components/ExternalReference.js

Would it be difficult to keep around a copy of the existing parts as-is? Given v3 is around the corner we'll be removing them then anyway, and it would be nice to to cause any breakage until then.

Reply:
Great pick up – and definitely not difficult to keep! That part removal has since been reverted and that original exported weighted search function can still be used as-is.